.evergreen/evergreen.yml.in

```diff
@@ -76,6 +76,579 @@ const platformDetails = platformToDetails[platform];
   }
 }
 
+const { RELEASE_PACKAGE_MATRIX } = require('../config/release-package-matrix');
+
+const ALL_E2E_TEST_BUILD_VARIANTS =  [
```

Comment:
Can we rename this to something that reflects that it doesn't just list e2e test variants, or put the compile variants into its own array (which is probably preferable anyway, because `tags` and `mVersion` probbaly don't make sense for them anyway)?